src/components/CreatePost/CreatePost.tsx

```diff
@@ -1,15 +1,30 @@
 import React, { useContext, useState } from 'react';
-import { FormControl, FormLabel, Input, Button } from '@chakra-ui/react';
-import { Form, Formik, Field, FieldProps, ErrorMessage } from 'formik';
-import * as Yup from 'yup';
-import { FirestoreContext } from '../../services/firestore/firestore';
+import {
+  FormControl,
+  Input,
+  Stack,
+  VStack,
+  Button,
+  Flex,
+  Spacer,
+  Box,
+  Textarea,
+  Text,
+  IconButton,
+} from '@chakra-ui/react';
+import { CloseIcon } from '@chakra-ui/icons';
+import { FirebaseContext } from '../../services/firebase/firebase';
+import ImageSlider, { ImagesInterface } from '../ImageSlider/ImageSlider';
```

Comment:
Better name would be ModifiableImageList, since it's not a slider anymore, The interface could be ModifiableImageListProps.